src/ServiceControl/Infrastructure/Settings/Settings.cs

```diff
@@ -336,7 +328,7 @@ private static bool GetForwardAuditMessages()
 
         static string GetTransportType()
         {
-            var typeName = SettingsReader<string>.Read("TransportType", typeof(MsmqTransport).AssemblyQualifiedName);
+            var typeName = SettingsReader<string>.Read("TransportType", "ServiceControl.Transports.Msmq.MsmqTransportCustomization, ServiceControl.Transports.Msmq");
```

Comment:
I wonder if we should get rid of the default value here. If we leave it then SC actually has to reference that customization because Type.GetType below would fail otherwise. If we remove it then you can't start SC.exe anymore without first copying a transport customization into the bin folder as well as explicitly setting a transport type though